tests/index.ts

```diff
@@ -0,0 +1,165 @@
+import * as fs from "fs";
+import * as path from "path";
+import { extractLandingState, ISetupSettings, ITestTarget, TrunkDriver } from "tests/driver";
+import { ILinterVersion, ITestingArguments } from "tests/types";
+
+let printed_args = false;
+
+// Parse the environment variable-specified linter version. This can either be
+// - KnownGoodVersion, which parses the linter definition and attempts to specify a known_good_version
+// - Latest, which automatically retrieves the latest linter version if network connectivity is available
+// - A specified version. Note that this will apply to all tests, so only use this environment variable when tests are filtered
+const parseLinterVersion = (value: string): ILinterVersion | string | undefined => {
+  if (value == "KnownGoodVersion") {
+    return ILinterVersion.KnownGoodVersion;
+  } else if (value == "Latest") {
+    return ILinterVersion.Latest;
+  } else if (value && value.length > 0) {
+    return value;
+  }
+  return undefined;
+};
+
+// Parse the global testing config inputs, specified as environment variables.
+// - PLUGINS_TEST_CLI_VERSION replaces the repo-wide trunk.yaml's specified cli-version
+// - PLUGINS_TEST_CLI_PATH specifies an alternative path to a trunk binary
+// - PLUGINS_TEST_LINTER_VERSION specifies a linter version semantic (see 'parseLinterVersion')
+export const parseInputs = () => {
```

Comment:
Unfortunately, passing custom command line arguments to jest requires some hacky approaches. Environment variables seems like the better approach for now.